addons/l10n_latam_check/views/account_payment_view.xml

```diff
@@ -0,0 +1,214 @@
+            </filter>
+            <filter name="journal" position="after">
+                <filter name="groupby_third_check_current_journal" string="Third Check Current Journal" domain="[]"
+                        context="{'group_by': 'l10n_latam_check_current_journal_id'}"/>
+            </filter>
+            <filter name="groupby_date" position="after">
+                <filter string="Check Payment Date" name="groupby_l10n_latam_check_payment_date" domain="[]" context="{'group_by': 'l10n_latam_check_payment_date'}"/>
+            </filter>
+        </field>
+    </record>
+
+    <record id="view_account_payment_form_inherited" model="ir.ui.view">
+        <field name="name">account.payment.form.inherited</field>
+        <field name="model">account.payment</field>
+        <field name="inherit_id" ref="account.view_account_payment_form" />
+        <field name="arch" type="xml">
+            <sheet position="before">
+                <div class="alert alert-warning mb-0" role="alert" attrs="{'invisible': [('l10n_latam_check_warning_msg', '=', False)]}">
+                    <field name="l10n_latam_check_warning_msg" nolabel="1"/>
+                </div>
+            </sheet>
+            <field name="check_number" position="attributes">
+                <attribute name="attrs">{'invisible': ['|', ('payment_method_code', '!=', 'check_printing'), ('check_number', '=', False), ('l10n_latam_checkbook_id', '=', False)], 'readonly': ['|', ('state', '!=', 'draft'), ('l10n_latam_checkbook_id', '=', False)], 'required': [('l10n_latam_checkbook_id', '!=', False)]}</attribute>
+            </field>
+
+            <xpath expr="//div[@name='amount_div']//field[@name='amount']" position="attributes">
+                <attribute name="attrs">{'readonly': ['|', ('state', '!=', 'draft'), ('payment_method_code', 'in', ['in_third_checks', 'out_third_checks'])]}</attribute>
+                <attribute name="force_save">1</attribute>
+            </xpath>
+            <field name="destination_journal_id" position="after">
+                <!-- Move Third Checks -->
+                <field
+                    name="l10n_latam_check_id"
+                    attrs="{'invisible': [('payment_method_code', 'not in', ['in_third_checks', 'out_third_checks']), ('l10n_latam_check_id', '=', False)],
+                            'required': [('payment_method_code', 'in', ['in_third_checks', 'out_third_checks'])]}"
+                    domain="[('payment_method_code', '=', 'new_third_checks'), ('l10n_latam_check_current_journal_id', '=', journal_id), ('state', '=', 'posted')] if payment_type == 'outbound' else [('payment_method_code', '=', 'new_third_checks'), ('l10n_latam_check_current_journal_id', '=', destination_journal_id), ('state', '=', 'posted')] if is_internal_transfer else [('payment_method_code', '=', 'new_third_checks'), ('l10n_latam_check_current_journal_id', '=', False), ('state', '=', 'posted')]"
```

Comment:
It was like that before (a computed field) but you ask to change it so that we avoid returning a list of available checks. It's working as it is now. But yes, a computed field returning checks would be more flexible if we need any adjustment

Reply:
done